core-charter.txt

```diff
@@ -1,40 +1,65 @@
+* use of CoAP to transport DNS queries and responses will also be investigated. 
 
-The CoRE WG originally selected DTLS as the basis for the communication security in CoAP. The CoRE WG will work with the TLS WG on the efficiency of this solution. The preferred cipher suites will evolve in cooperation with the TLS WG and CFRG.
+* enabling broker-based publish-subscribe-style communication over CoAP
 
-The CoRE WG considered object security as originally defined in JOSE and later adapted to the constrained node network requirements in COSE. Building on that, the CoRE WG has defined the Object Security for Constrained RESTful Environments (OSCORE) protocol (RFC 8613) for protecting CoAP messages at the application layer using COSE. The CoRE WG will complete the work on the Group OSCORE protocol (draft-ietf-core-oscore-groupcomm) that enables OSCORE-protection of CoAP messages in group communication environments. Also, the CoRE WG will complete an optimization-oriented profiling of the EDHOC protocol developed in the LAKE WG, when this is used to establish security material for OSCORE (draft-ietf-core-oscore-edhoc). Furthermore, the CoRE WG will perform maintenance as well as possible updating and complementing of the (Group) OSCORE documents above as required.
+* related data formats, such as alternative representations of RFC 6690 link
 
-The ACE WG is expected to provide solutions on authentication and authorization that may need complementary elements on the CoRE side.
+* Constrained Resource Identifiers for serializing URI components in CBOR
 
-The CoRE WG will coordinate on requirements from many organizations and SDOs. The CoRE WG will closely coordinate with other IETF WGs, particularly of the constrained node networks cluster (6Lo, 6TiSCH, LWIG, ROLL, ACE, CBOR, COSE, IOTOPS, LAKE, SUIT), and with further appropriate groups in the IETF OPS and Security Areas. Work on these subjects, as well as on data/interaction models and design patterns (including follow-up work around the CoRE Interfaces draft) will additionally benefit from close cooperation with the Thing-to-Thing Research Group.
+* develop CoRAL, a constrained RESTful application language intended for driving automated software agents that navigate a Web application
+
+* complete the Sensor Measurement Lists (SenML) set of specifications
+
+* continuing to examine operational and manageability aspects of the CoAP protocol itself
+
+* complete the work on RESTCONF-style management functions available via CoAP that are appropriate for constrained node networks 
+
+* efficiency of DTLS as the basis for the communication security in CoAP
```

Comment:
I don't know what this work item is, but I definitely don't feel it belongs here.